proposed/code-of-conduct.md

```diff
@@ -0,0 +1,101 @@
+# Code of Conduct for the MODX Project
+
+Note: This recommendation is a DRAFT for consideration of the MODX Advisory Board. 
+
+A Code of Conduct is a document that details how individuals are expected to behave in a certain context. By providing a baseline for interactions, and a process for dealing with incidents, we can ensure everyone feels welcomed to the community. 
+
+Editor: Mark Hamstra
+
+First published draft: Not yet reviewed/published.
+
+## Goal of Recommendation
+
+The recommendation serves to ensure a welcoming environment in the online and offline MODX community. This is accomplished through two aspects of this recommendation:
+
+- The Code of Conduct document, based on the Contributor Covenant, which contains the baseline for respect and inclusivity that we expect from everyone involved with the MODX community.
+- A process definition and guidelines that explain how the MODX Advisory Board responds to reports of breaches of the Code of Conduct.  
+
+## Relevant Recommendations
+
+None.
+
+## Recommendation
+
+### 1. The Code of Conduct
+
+The text for the proposed Code of Conduct can be found in Attachment 1. It is based on the [Contributor Covenant 1.4.0](http://contributor-covenant.org/version/1/4/), which is a widely used Code of Conduct and should be considered included verbatim in this recommendation. 
```

Comment:
Thomas Gautveldt made a comment on the drive version of this document that it didn't include the actual code of conduct text. In response to that I've tweaked this section a bit, however I'm considering actually including it in the recommendation so it can be part of the discussion.